.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,93 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. For more details see the
+        [Contributing Guidelines][contributing-guidelines].
+
+
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `new icon` label][new-icon-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction.
+
+
+        [contributing-guidelines]:
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+
+        [new-icon-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+      description: >-
+        For non-web brands you can add a relevant link. You can put "None" if you don't
+        think there's a website.
+      placeholder: "Example: https://simpleicons.org"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Popularity Metric
+      description: >-
+        Provide either an [Alexa rank][alexa-site-info] or the number of GitHub stars.
+        If these are not available or too low you can still open the issue. Please
+        provide any information regarding the brand's popularity you think is relevant
+        below.
+
+
+        [alexa-site-info]: https://www.alexa.com/siteinfo
+      placeholder: "Example: 249,089"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. You can set
+        this to none if you are unable to find any resources.
+      placeholder: |
+        Example:
+        svg source: https://github.com/simple-icons/simple-icons/blob/develop/icons/simpleicons.svg
+        color: `#111111`, from the website header
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional Comments
+      description: |
+        Is there anything else we should know about the brand? Remember that not everyone knows the brand as well as you do. For example:
+        - Are there multiple options for the logo and/or color?
+        - Is the icon released under a license?
+        - If the brand isn't popular enough by the default criteria, how can we assess the popularity instead?
+        - If you think the brand might not be accepted for other reasons, why do you think it should be considered?
```

The first one should be removed with the update to the "Popularity metric" input, and as a result the second one should be reworded, maybe:

```suggestion
        - If you think the brand might not be accepted, why do you think it should be considered?
```